src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Tasklist object that allows abstraction to track and edit the list of tasks.
+ */
+public class TaskList {
+    /**
+     * The list of tasks.
+     */
+    private final ArrayList<Task> TASKS;
```

Comment:
Similar comment as above, perhaps your intention was to use private static final for this naming convention.

Reply:
Okay